lib/getReactFlowData.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generates reactFlow elements data from default output syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default output syntax
+ * @returns {Array} reactFlow elements data following 
+ */
+
+function getReactFlowData(metrics) {
+  let i=1;
+  let edgId = 1;
+  const elements = [];
```

Comment:
```suggestion
  const nodes = [];
```

Will be better name :)